src/Card.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+#include <string>
+#include <array>
+
+class Card
+{
+    std::array<std::string, 16> jobs_;
```

Comment:
I'm not sure if I get a concept of this representation. We will have an array with cards descriptions, which are set by a setter. Why not in a constructor? Who (which class) will set those values?